README.md

```diff
@@ -2,7 +2,8 @@
 
 [![Syntax Check](https://travis-ci.org/common-workflow-language/user_guide.svg?branch=main)](https://travis-ci.org/common-workflow-language/user_guide)
 
-[User guide for CWL v1.2.0](https://www.commonwl.org/user_guide/)
+[Released User guide for CWL v1.2.0](https://www.commonwl.org/user_guide/) : This is the released version which corresponds to the release branch.  
+[Read The Docs User guide for CWL v1.2.0](https://common-workflow-languageuser-guide.readthedocs.io/en/latest/) : This is the Read The Docs(RTD) version which corresponds to the main branch.
```

Comment:
Use plain URL links here, please. The link text is too similar to each other.

`RTD` is defined here and never used again, so remove that abbreviation.

Reply:
I believe it was added because we in tickets and other places (like chat) use RTD so people were wondering what that meant.